neurodamus/ngv.py

```diff
@@ -1,5 +1,8 @@
 """Module which defines and handles Glia Cells and connectivity"""
 
+# TODO(cattabiani): in https://github.com/openbraininstitute/neurodamus/pull/266
```

Comment:
my point was to put it in the docs like Endoplasmic Reticulum here:

https://github.com/openbraininstitute/neurodamus/blob/main/docs/source/ngv.rst